src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerWaitContainer.groovy

```diff
@@ -16,9 +16,17 @@
 package com.bmuschko.gradle.docker.tasks.container
 
 class DockerWaitContainer extends DockerExistingContainer {
+
+    Integer exitCode;
+
+    DockerWaitContainer() {
+        ext.getExitCode = { exitCode }
+    }
+
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Waiting for container with ID '${getContainerId()}'."
-        dockerClient.waitContainerCmd(getContainerId()).exec()
+        exitCode = dockerClient.waitContainerCmd(getContainerId()).exec()
```

Comment:
It's very useful if you want to execute unit tests inside container. exitCode is easy way to determine if tests are passed or not.